koku/masu/database/aws_report_db_accessor.py

```diff
@@ -197,23 +198,26 @@ def populate_line_item_daily_table(self, start_date, end_date, bill_ids):
 
         """
         table_name = AWS_CUR_TABLE_MAP['line_item_daily']
+
+        jinjaSql = JinjaSql()
```

Comment:
This is a nitpick, but consistent with our code. Use lowercase where it makes sense and separate with underscores. 
```suggestion
        jinja_sql = JinjaSql()
```